addons/survey/controllers/main.py

```diff
@@ -426,7 +426,7 @@ def survey_submit(self, survey_token, answer_token, **post):
         return json.dumps(ret)
 
     @http.route('/survey/print/<string:survey_token>', type='http', auth='public', website=True)
-    def survey_print(self, survey_token, answer_token=None, **post):
+    def survey_print(self, survey_token, review=None, answer_token=None, **post):
```

Comment:
as review is used as a Boolean, set default value to False.

Reply:
It's done. Thanks